### PR TITLE
Add ING'Homepay icon

### PR DESCRIPTION
**Pull request by vernondegoede**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by vernondegoede** [comment]
@tauthomas01 My bad, I didn't properly check the test results.

Tests are now passing. Could you take another look?

File: app/assets/images/payment_icons/ing-homepay.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" aria-labelledby="pi-ing-homepay" viewBox="0 0 38 24" role="img">
+  <title id="pi-ing-homepay">ING Home'Pay</title>
+  <defs/>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/ing-homepay.svg]: This is not needed

File: app/assets/images/payment_icons/ing-homepay.svg [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  <path fill="#FF5300" d="M34 6h4v8h-4z"/>
+  <use fill="transparent" stroke="rgba(0, 0, 0, 0.07)" stroke-miterlimit="10" stroke-width="2"/>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/ing-homepay.svg]: I don't see the transparent border. Could you and try to follow similar pattern like Shopify Pay?

https://github.com/activemerchant/payment_icons/blob/efd33be748023e67e7b25c7e483277247a13f604/app/assets/images/payment_icons/shopify_pay.svg

File: .gitignore [hunk 3]
```diff
@@ -7,3 +7,4 @@ test/dummy/tmp/
 test/dummy/.sass-cache
 gemfiles/*.lock
 .DS_Store
+.idea/
```
Comment (tauthomas01) [inline .gitignore]: Can you remove this?

**Pull request closed without merging**